crates/pathfinder/src/rpc.rs

```diff
@@ -252,6 +252,17 @@ pub async fn run_server(
         let params = params.parse::<NamedArgs>()?;
         context.call(params.request, params.block_hash).await
     })?;
+    module.register_async_method("starknet_estimateFee", |params, context| async move {
```

Comment:
The core of this method is tested in `ext_py.rs` but I was wondering if it would be easy to add a few tests to assert for the [error codes from the spec](https://github.com/starkware-libs/starknet-specs/blob/e4543d8a83cbd78d3ac08a2e2512e2a954fdcc00/api/starknet_api_openrpc.json#L502).

Btw I just realized that the tests for `call` in this file don't make sense cause the [`handle` is `None`](https://github.com/eqlabs/pathfinder/blob/24bf184ff55c65c9f3b45177278a3e8de5413bf7/crates/pathfinder/src/rpc/api.rs#L981).
We could either remove them, or better fix both (`estimate_fee` & `call`) to actually use the local cairo machinery in all those cases. Not in this very PR ofc.